### PR TITLE
Replace global variable with markRaw

### PR DESCRIPTION
**Pull request by ddjnw1yu**

### CODE REVIEW

File: src/components/LinesControls.vue [hunk 1]
```diff
@@ -179,50 +177,50 @@ export default {
...
+        this.zincObject = markRaw(object);
+        this.width = this.zincObject.getMorph().material.linewidth;
         if (object.isEditable) {
           this.currentIndex = 0;
           this.distance = getLineDistance(object, this.currentIndex);
         }
       } else {
-        this._zincObject = undefined;
+        this.zincObject = undefined;
         this.linewidth = 10;
```
Comment (ddjnw1yu) [inline src/components/LinesControls.vue:219]: A question: Should the `linewidth` here be `width`? Or is this a different variable?

File: src/components/ScaffoldTreeControls.vue [hunk 2]
```diff
@@ -117,8 +120,8 @@ export default {
     // '__r/'
     findOrCreateRegion: function (data, paths, prefix) {
       //check if root region has been set
-      if (this.rootID === undefined && this.$module && this.$module.scene) {
-        this.treeData[0].id = this.$module.scene.getRootRegion().uuid;
+      if (this.rootID === undefined && this.module && this.module.scene) {
```
Comment (ddjnw1yu) [inline src/components/ScaffoldTreeControls.vue:123]: Another question: Where does the `rootID` come from? I didn't find it in any other files.

File: src/components/PrimitiveControls.vue [hunk 3]
```diff
@@ -108,7 +109,7 @@ export default {
     },
     setObject: function(object) {
       if (object) {
-        this.zincObject = shallowRef(object);
+        this.zincObject = markRaw(object);
```
Comment (alan-wu) [inline src/components/PrimitiveControls.vue:112]: This one should remain as shallowRef, it is referenced in the template.

File: src/components/LinesControls.vue (hunk 1 above)
Comment (alan-wu) [reply src/components/LinesControls.vue:219]: It should be width. Thanks for catching that.

File: src/components/ScaffoldTreeControls.vue (hunk 2 above)
Comment (alan-wu) [reply src/components/ScaffoldTreeControls.vue:123]: Remove that line please,

Comment (alan-wu) [reply src/components/ScaffoldTreeControls.vue:123]: You can safely remove this. Thanks.

**Review by alan-wu — APPROVED** [review]
Looks good to me. I have made the small changes where you left the comment. Thanks.

**Pull request merged**